acct-user/mythtv/mythtv-0.ebuild

```diff
@@ -7,7 +7,7 @@ inherit acct-user
 
 DESCRIPTION="Mythtv mythbackend server/deamon user"
 ACCT_USER_ID=117
-ACCT_USER_GROUPS=( mythtv video audio tty )
+ACCT_USER_GROUPS=( mythtv video audio cdrom tty )
```

Comment:
Doesn't the ebuild needs to be revbumped, in order to get that changes installed on already pulled in systems?

Reply:
Your may be correct, but repoman does not accept the revision bump as stable. 

I added `KEYWORDS="~amd64 ~x86"` to get past repoman. I am not sure of the right way to do this, but the revision bump needs to be stable too. Both the changed original and revision bump fix the bug.

I make separate commits so that whoever merges this can decide what to do.

Reply:
If you add "-S" to repoman, it bumps strait to stable ;-)

Reply:
Thank you. It works.